lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -14,6 +14,39 @@ def self.email(params = {}, _secrets = {}, _context = {})
           miq_task_status!(runner_context)
         end
 
+        def self.embedded_ansible(params = {}, _secrets = {}, _context = {})
+          repository_url, repository_branch, playbook_name, playbook_id = params.values_at("RepositoryUrl", "RepositoryBranch", "PlaybookName", "PlaybookId")
+
+          vars = params
+                 .slice("Hosts", "ExtraVars", "BecomeEnabled", "Timeout", "Verbosity", "CredentialId", "CloudCredentialId", "NetworkCredentialId", "VaultCredentialId")
+                 .transform_keys { |k| k.underscore.to_sym }
+
+          vars[:execution_ttl] = vars.delete(:timeout) if vars.key?(:timeout)
+          %i[credential_id cloud_credential_id network_credential_id vault_credential_id].each do |key|
+            new_key = key.to_s.gsub(/_id$/, '').to_sym
```

Comment:
Not necessary to fix, but this can also be simplified to
```suggestion
            new_key = key.to_s.chomp("_id").to_sym
```